zng/docs/spec.md

```diff
@@ -763,7 +687,36 @@ An unset value indicates a field of a `record` that wasn't set by the encoder:
 ```
 e.g., the North Pole has a latitude but no meaningful longitude.
 
-## 5. Related Links
+## 5. Data Type Reference
```

Comment:
This section deals exclusively with primitive types, and it doesn't just serve as a reference—it defines them—so I'd title it "Primitive Types".

Reply:
@nwt: That's a good suggestion. I've pushed a commit to make that adjustment in the section header as well as the TOC and the three places in the spec that link to here.